docs/ovs-pipeline.md

```diff
@@ -574,40 +577,47 @@ non-ARP and non-IP (assuming any can be received by the switch) is actually
 dropped much earlier in the pipeline ([SpoofGuardTable]). In the future, we may
 need to support more cases for L2 multicast / broadcast traffic.
 
-### CnpIngressRuleTable (85)
+### CnpIngressRuleTables (85-89)
 
-This table is very similar to [CnpEgressRuleTable], but implements the ingress
-rules of the CNP. Again for this table, you will need to keep mind the CNP
-[specification](#antrea-network-policy-crd-implementation) that we are using.
+These tables are very similar to [CnpEgressRuleTables], but implement the ingress
+rules of the CNP. Depending on the tier to which the CNP belongs to, the rules will
+be installed in a table corresponding to that tier. The ingress CNP table to tier
+mappings is as follows:
 
-If you dump the flows for this table, you should see something like this:
 ```
-1. table=85, priority=64990,ct_state=-new+est,ip actions=resubmit(,105)
-2. table=85, priority=11800,conj_id=1,ip actions=drop
-3. table=85, priority=11800,ip,nw_src=10.10.1.7 actions=conjunction(1,1/3)
-4. table=85, priority=11800,ip,reg1=0x19c actions=conjunction(1,2/3)
-5. table=85, priority=11800,tcp,tp_dst=80 actions=conjunction(1,3/3)
-6. table=85, priority=0 actions=resubmit(,90)
+1. Emergency tier -> EmergencyIngressRuleTable(85)
+2. SecurityOps tier -> SecurityOpsIngressRuleTable(86)
+3. NetworkOps tier -> NetworkOpsIngressRuleTable(87)
+4. Platform tier -> PlatformIngressRuleTable(88)
+5. Application tier -> ApplicationIngressRuleTable(89)
 ```
 
-As for [CnpEgressRuleTable], flow 1 (highest priority) ensures that for
+Again for this table, you will need to keep in mind the CNP
+[specification](#antrea-network-policy-crd-implementation) that we are using.
+Since the example CNP resides in the Application tier, if you dump the flows
+for table 89, you should see something like this:
+```
+1. table=89, priority=64990,ct_state=-new+est,ip actions=resubmit(,105)
+2. table=89, priority=11800,conj_id=1,ip actions=drop
+3. table=89, priority=11800,ip,nw_src=10.10.1.7 actions=conjunction(1,1/3)
+4. table=89, priority=11800,ip,reg1=0x19c actions=conjunction(1,2/3)
+5. table=89, priority=11800,tcp,tp_dst=80 actions=conjunction(1,3/3)
+6. table=89, priority=0 actions=resubmit(,90)
+```
+
+As for [CnpEgressRuleTables], flow 1 (highest priority) ensures that for
 established connections packets go straight to [L2ForwardingOutTable],
 with no other match required.
 
 The rest of the flows read as follows: if the source IP address is in set
 {10.10.1.7}, and the destination OF port is in the set {412} (which
 correspond to IP addresses {10.10.1.6}), and the destination TCP port
 is in the set {80}, then use `conjunction` action with id 1, which drops the
-packet. Otherwise, go to [IngressRuleTable]. One notable difference is how we
```

Comment:
it's a bit strange because this section comes before the K8s NP one, yet we are removing ti

Reply:
updated